pkg/route/types.go

```diff
@@ -104,8 +104,11 @@ func (rc *Config) Validate(ctx context.Context) error {
 	if rc.Id == "" {
 		return errors.New("missing ID for plugin configuration")
 	}
+	validName := regexp.MustCompile(`^[a-zA-Z][a-zA-Z0-9_\-\.]*$`)
```

Comment:
do we want to limit the (_, -, and .) to only the middle of the string? I think the current expression will validate even if the characters are the first/last charaster

Reply:
ok, will make that change